### ISSUE TITLE
Background refresh of Archive

### ISSUE THREAD

**Issue by awlx** [buttercup/buttercup-browser-extension#170]
It seems the Browser extension for Chrome does not automatically refresh the Archive. If I change something through the Desktop App or Mobile it does never appear in the Browsers suggestions.

 I have to manually lock the Archive and reopen it for new entries to appear.

**Comment by perry-mitchell** [comment]
Blocked by buttercup/buttercup-core#200

**Comment by perry-mitchell** [comment]
Also duplicate of #110
